### ISSUE TITLE
Compile separate stylesheets into a single one rather than using @imports

### ISSUE THREAD

**Issue by sleicester** [the-open-university/ouice#19]
_From @sleicester on December 5, 2014 8:46_

This saves web requests.
We can also provide a minified version, although this does make it harder to debug and considering it's cached.

_Copied from original issue: the-open-university/student-theme-framework#19_

**Comment by stuartcrouch** [comment]
Look up css sourcemaps. I can't think what situation you wouldn't be able to debug, especially with a CDN storing revisions for you.

**Comment by bgurney** [comment]
I think some of these comments are quite old

**Comment by sleicester** [comment]
Yeah, we've used [Laravel Elixir](https://laravel.com/docs/5.2/elixir) as a nice wrapper around Gulp for a lot of these tasks, that provides nice source maps.

You can use it outside of Laravel as well, it's really a node package.

**Comment by stuartcrouch** [comment]
Sorry for the spam. Just got access and saw all the comments :)
